packages/unified-latex-to-pretext/tests/unified-latex-to-pretext.test.ts

```diff
@@ -164,29 +177,49 @@ describe("unified-latex-to-pretext:unified-latex-to-pretext", () => {
             `\\begin{itemize}before content\\item[x)] a\\item[] b\\end{itemize}`
         );
         expect(normalizeHtml(html)).toEqual(
-            normalizeHtml(`<ul class="itemize">
-                <li style="list-style-type: &#x27;x) &#x27;"><p>a</p></li>
-                <li style="list-style-type: none;"><p>b</p></li>
-            </ul>`)
+            // normalizeHtml(`<ul class="itemize">
+            //     <li style="list-style-type: &#x27;x) &#x27;"><p>a</p></li>
+            //     <li style="list-style-type: none;"><p>b</p></li>
+            // </ul>`)
+            normalizeHtml(
+                `<dl>
+                    <li><title>x)</title><p>a</p></li>
+                    <li><title></title><p>b</p></li>
+                </dl>` // markers are bolded and list is centered though
+            )
         );
     });
 
     it.skip("Converts tabular environment", () => {
         html = process(`\\begin{tabular}{l l}a & b\\\\c & d\\end{tabular}`);
         expect(normalizeHtml(html)).toEqual(
+            // normalizeHtml(
+            //     `<table class="tabular">
+            //     <tbody>
+            //         <tr>
+            //             <td>a</td>
+            //             <td>b</td>
+            //         </tr>
+            //         <tr>
+            //             <td>c</td>
+            //             <td>d</td>
+            //         </tr>
+            //     </tbody>
+            // </table>`
+            // )
+
+            // centered tho
             normalizeHtml(
-                `<table class="tabular">
-                <tbody>
-                    <tr>
-                        <td>a</td>
-                        <td>b</td>
-                    </tr>
-                    <tr>
-                        <td>c</td>
-                        <td>d</td>
-                    </tr>
-                </tbody>
-            </table>`
+                `<tabular>
+                    <row>
+                        <cell>a</cell>
+                        <cell>b</cell>
+                    </row>
+                    <row>
+                        <cell>c</cell>
+                        <cell>d</cell>
+                    </row>
+                </tabular>`
```

Comment:
It'd be good to have a test of `\begin{tabular}{r l} ...` where cells in the first column got `halign="right"` and the second column was left as its default value (of `halign="left"`)